pkg/apiserver/resthandler.go

```diff
@@ -589,6 +589,7 @@ func patchResource(ctx api.Context, admit updateAdmissionFunc, timeout time.Dura
 				return nil, err
 			}
 			if hasConflicts {
+				glog.V(2).Infof("patchResource failed becauase there is a meaningful conflict.\n diff1=%v\n, diff2=%v\n", diff1, diff2)
```

Comment:
Maybe V(4).

Also, include more info about the resource the patch is trying to operate on, so we can tell which request the message is for when reading the logs.